packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -426,15 +437,21 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
     ServicesBinding.instance!.restorationManager.flushData();
   }
 
+  @override
+  void initState() {
+    _scrollController = ScrollController();
```

Comment:
We should probably only create this one if widget.controller is null. In `didUpdateWidget` we will have to dispose/create the ScrollController if `widget.controller` changes.